From 33a3942971f3f56fb4b70804ec13b15ee2254caa Mon Sep 17 00:00:00 2001
From: Andrej Rode <andrej.rode@ettus.com>
Date: Mon, 27 Feb 2017 14:23:56 -0800
Subject: [PATCH 14/40] uhd: Fix some log statements

Some log statements were incorrectly using multi-line log statements.
---
 host/lib/rfnoc/block_ctrl_base.cpp     | 1 -
 host/lib/usrp/b200/b200_impl.cpp       | 4 ++--
 host/lib/usrp/device3/device3_impl.cpp | 1 -
 host/lib/usrp/x300/x300_impl.cpp       | 6 +++---
 4 files changed, 5 insertions(+), 7 deletions(-)

diff --git a/host/lib/rfnoc/block_ctrl_base.cpp b/host/lib/rfnoc/block_ctrl_base.cpp
index bc2c2a47f..3bf0edd87 100644
--- a/host/lib/rfnoc/block_ctrl_base.cpp
+++ b/host/lib/rfnoc/block_ctrl_base.cpp
@@ -253,7 +253,6 @@ void block_ctrl_base::sr_write(const std::string &reg, const uint32_t data, cons
         }
         reg_addr = uint32_t(_tree->access<size_t>(_root_path / "registers" / "sr" / reg).get());
     }
-    UHD_BLOCK_LOG() << "  ";
     UHD_RFNOC_BLOCK_TRACE() << boost::format("sr_write(%s, %08X) ==> ") % reg % data ;
     return sr_write(reg_addr, data, port);
 }
diff --git a/host/lib/usrp/b200/b200_impl.cpp b/host/lib/usrp/b200/b200_impl.cpp
index 4a96b514b..570994496 100644
--- a/host/lib/usrp/b200/b200_impl.cpp
+++ b/host/lib/usrp/b200/b200_impl.cpp
@@ -886,7 +886,7 @@ void b200_impl::setup_radio(const size_t dspno)
 void b200_impl::register_loopback_self_test(wb_iface::sptr iface)
 {
     bool test_fail = false;
-    UHD_LOGGER_INFO("B200") << "Performing register loopback test... " << std::flush;
+    UHD_LOGGER_INFO("B200") << "Performing register loopback test... ";
     size_t hash = size_t(time(NULL));
     for (size_t i = 0; i < 100; i++)
     {
@@ -895,7 +895,7 @@ void b200_impl::register_loopback_self_test(wb_iface::sptr iface)
         test_fail = iface->peek32(RB32_TEST) != uint32_t(hash);
         if (test_fail) break; //exit loop on any failure
     }
-    UHD_LOGGER_INFO("B200") << ((test_fail)? "fail" : "pass") ;
+    UHD_LOGGER_INFO("B200") << "Register loopback test " << ((test_fail)? "failed" : "passed") ;
 }
 
 /***********************************************************************
diff --git a/host/lib/usrp/device3/device3_impl.cpp b/host/lib/usrp/device3/device3_impl.cpp
index 6304b5df4..7792a4b73 100644
--- a/host/lib/usrp/device3/device3_impl.cpp
+++ b/host/lib/usrp/device3/device3_impl.cpp
@@ -125,7 +125,6 @@ void device3_impl::enumerate_rfnoc_blocks(
                 xport.send_sid,
                 str(boost::format("CE_%02d_Port_%02X") % i % ctrl_sid.get_dst_endpoint())
         );
-        UHD_DEVICE3_LOG() << "OK" ;
         uint64_t noc_id = ctrl->peek64(uhd::rfnoc::SR_READBACK_REG_ID);
         UHD_DEVICE3_LOG() << str(boost::format("Port %d: Found NoC-Block with ID %016X.") % int(ctrl_sid.get_dst_endpoint()) % noc_id) ;
         uhd::rfnoc::make_args_t make_args;
diff --git a/host/lib/usrp/x300/x300_impl.cpp b/host/lib/usrp/x300/x300_impl.cpp
index 61b65eec0..97c8c0682 100644
--- a/host/lib/usrp/x300/x300_impl.cpp
+++ b/host/lib/usrp/x300/x300_impl.cpp
@@ -443,7 +443,7 @@ void x300_impl::mboard_members_t::discover_eth(
         if (std::find(mb_eeprom_addrs.begin(), mb_eeprom_addrs.end(), mb_eeprom[key]) != mb_eeprom_addrs.end()) {
             UHD_LOGGER_WARNING("X300") << str(boost::format(
                 "Duplicate IP address %s found in mboard EEPROM. "
-                "Device may not function properly.\nView and reprogram the values "
+                "Device may not function properly. View and reprogram the values "
                 "using the usrp_burn_mb_eeprom utility.") % mb_eeprom[key]);
         }
         mb_eeprom_addrs.push_back(mb_eeprom[key]);
@@ -473,8 +473,8 @@ void x300_impl::mboard_members_t::discover_eth(
         if (conn_iface.type == X300_IFACE_NONE) {
             UHD_LOGGER_WARNING("X300") << str(boost::format(
                 "Address %s not found in mboard EEPROM. Address may be wrong or "
-                "the EEPROM may be corrupt.\n Attempting to continue with default "
-                "IP addresses.\n") % conn_iface.addr
+                "the EEPROM may be corrupt. Attempting to continue with default "
+                "IP addresses.") % conn_iface.addr
             );
 
             if (addr == boost::asio::ip::address_v4(
-- 
2.11.0

